requirements-dev.txt

```
@@ -3,13 +3,10 @@
 # pip install -r requirements.txt
 
 chromedriver-binary
-dash>=0.31.0
-dash-core-components
-dash-html-components
-dash-renderer
+dash==2.6.2
```

```suggestion
dash[ci,dev,testing]==2.6.2
```

And delete the then-redundant `dash[testing]` below.
- `[ci]` includes a working `black` among other things.
- `[dev]` for building per my earlier comment

---

Fixed